src/ResourceManager/DataLakeStore/Commands.DataLakeStore/packages.config

```diff
@@ -1,4 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
+  <package id="Microsoft.Azure.DataLake.Store" version="1.0.2" targetFramework="net452" />
   <package id="Microsoft.Azure.Management.DataLake.Store" version="2.3.0-preview" targetFramework="net452" />
+  <package id="NLog" version="4.4.12" targetFramework="net452" />
```

Comment:
@rahuldutta90 since we've confirmed that these packages are OK to use, you will need to add them to the `RequiredAssemblies` field in the [`AzureRM.DataLakeStore.psd1`](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/DataLakeStore/AzureRM.DataLakeStore.psd1#L57) file.